Source/Diagnostics/BoundaryScrapingDiagnostics.H

```diff
@@ -0,0 +1,63 @@
+/* Copyright 2022 Remi Lehe
+ *
+ * This file is part of WarpX.
+ *
+ * License: BSD-3-Clause-LBNL
+ */
+#ifndef WARPX_BOUNDARYSCRAPINGDIAGNOSTICS_H_
+#define WARPX_BOUNDARYSCRAPINGDIAGNOSTICS_H_
+
+#include "Diagnostics.H"
+
+#include <string>
+
+/** collect the particles that are absorbed at the embedded boundary, throughout the simulation
+ */
+class
+BoundaryScrapingDiagnostics final : public Diagnostics
+{
+public:
+
+    /** Constructor
+     *
+     * @param i ...
+     * @param name ...
```

Comment:
What's `i` and `name` again, I forgot...
```suggestion
     * @param i ... ?
     * @param name diagnostics name in the inputs file
```